storage/storage_helpers.cpp

```diff
@@ -18,6 +20,17 @@ bool IsDownloadFailed(Status status)
          status == Status::EUnknown;
 }
 
+bool IsEnoughSpaceForDownload(TCountryId const & countryId, Storage const & storage)
+{
+  NodeAttrs nodeAttrs;
+  storage.GetNodeAttrs(countryId, nodeAttrs);
+  size_t constexpr kDownloadExtraSpaceSize = 100 /*Mb*/ * 1024 * 1024;
```

Comment:
Может это достойно комментария в коде?

> On Apr 1, 2016, at 11:36, Alexander Marchuk <notifications@github.com> wrote:
> 
> In storage/storage_helpers.cpp <https://github.com/mapsme/omim/pull/2665#discussion_r58176930>:
> 
> > @@ -18,6 +20,17 @@ bool IsDownloadFailed(Status status)
> >           status == Status::EUnknown;
> >  }
> >  
> > +bool IsEnoughSpaceForDownload(TCountryId const & countryId, Storage const & storage)
> > +{
> > +  NodeAttrs nodeAttrs;
> > +  storage.GetNodeAttrs(countryId, nodeAttrs);
> > +  size_t constexpr kDownloadExtraSpaceSize = 100 /*Mb*/ * 1024 * 1024;
> Вот, теперь я допёр, спасибо)
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2665/files/fd0387d3b448a6c4cce4d03e1d5b428f2dd3c53f#r58176930>